GPU/Common/ShaderUniforms.h

```diff
@@ -43,6 +39,10 @@ struct UB_VS_FS_Base {
 	float blendFixB[4];
 	float texClamp[4];
 	float texClampOffset[4];
+	int spline_count_u;
+	int spline_count_v;
+	int spline_type_u;
+	int spline_type_v;
```

Comment:
Right, I forgot we had that split. Didn't really mean to include that move anyway, I'll move them back.